src/main.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+import './style/main.scss';
+
+const apiUrl = 'http://www.reddit.com/r/';
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchFormBoard: '',
+      searchFormLimit: '',
+    };
+
+    this.handleRedditBoardChange = this.handleRedditBoardChange.bind(this);
+    // this.handleSubmit = this.handleSubmit.bind(this);
+    // this.handleRedditLimitChange = this.handleRedditLimitChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleRedditBoardChange(event) {
+    this.setState({
+      searchFormBoard: event.target.value,
+    });
+  }
+
+  // handleRedditLimitChange(event) {
+  //   console.log(event.target.value);
+  //   this.setState({
+  //     renderTopicsList: event.target.value,
+  //   });
+  // }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.redditSelect(this.state.searchFormBoard);
+    // this.props.renderTopicsList(this.state.renderTopicsList);
```

Comment:
add a second argument to pass into this function with the limit created from the input field